packages/react-sample/package.json

```diff
@@ -5,15 +5,17 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "precommit": "lint-staged",
-    "start": "parcel src/index.html",
+    "start": "parcel src/index.html -p 3000",
```

Comment:
will it be better to align parcel port usage across the reops? `PORT=3000 parcel src/index.html`

Reply:
I'm OK with either way. But I just noticed we have other parcel options specified in `logto/console`
```
"dev": "PORT=5002 parcel src/index.html --public-url /console --no-hmr --no-cache",
```
Would it be better using `-p` or `--port` to align with other option usages?

Reply:
ok, after a quick heads-up, we decided to use `-p` across all repos. Will create another PR to address this